src/components/NoteFormSocial.js

```diff
@@ -0,0 +1,150 @@
+import React, {useEffect, useState} from "react";
+import Button from "../styles/Button";
+import {RenderedNote} from "../styles/TextArea";
+import Card from "./Card";
+import {FlexCenter} from "../styles/Grid";
+import {PencilIcon} from "@primer/octicons-react";
+import ReactMarkdown from "react-markdown";
+import api from "../lib/apiGraphQL";
+import {AllStyledComponent} from "@remirror/styles/emotion";
+import {CodeExtension} from "@remirror/extension-code";
+import {BulletListExtension} from "@remirror/extension-list";
+import {HeadingExtension} from "@remirror/extension-heading";
+import {BoldExtension} from "@remirror/extension-bold";
+import {ItalicExtension} from "@remirror/extension-italic";
+import {TableExtension} from "@remirror/extension-react-tables";
+import {MarkdownExtension} from "@remirror/extension-markdown";
+import {EmojiExtension} from "@remirror/extension-emoji";
+import {EditorComponent, ThemeProvider, Remirror, useRemirror, useCommands, useEmoji} from "@remirror/react";
+import {EmojiPopupComponent} from "@remirror/react-components";
+import emojiData from "svgmoji/emoji-github";
+const extensions = () => [
+  new CodeExtension(),
+  new BulletListExtension(),
+  new HeadingExtension(),
+  new BoldExtension(),
+  new ItalicExtension(),
+  new TableExtension(),
+  new MarkdownExtension(),
+  new EmojiExtension({
+    data: emojiData,
+    moji:"noto",
+    fallback:":-)",
+    suggestionCharacter:":",
+    plainText:true
+  }),
+];
+const EmojiEditor = () => {
+  useEmoji();
+  return (
+    <EditorComponent />
+  );
+};
+const Menu = () => {
+  // Access the commands and the activity status of the editor.
+  const commands = useCommands();
+  const startNewNote = () => {
+    commands.insertMarkdown("### Notes for Repo\n- fork the repo\n\n- follow everybody on twitter\n-troll, troll, troll<Cursor>");
```

Comment:
@bdougie this is a bit of an easter egg. anyway, I'm undecided about whether to keep this button in place (@0-vortex had suggested changing the label on it), as opposed to just pre-loading some template markdown when someone starts editing a blank note.

Reply:
freedom!

Reply:
Let's name the button, "basic template"

Reply:
👍